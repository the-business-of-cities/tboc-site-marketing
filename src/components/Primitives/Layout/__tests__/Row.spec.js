import { Row } from "../index";
import React from "react";
import { shallow } from 'enzyme';

describe("Row", () => {
  it("renders correctly", () => {
    expect(shallow(<Row />)).toMatchSnapshot();
  });
});
