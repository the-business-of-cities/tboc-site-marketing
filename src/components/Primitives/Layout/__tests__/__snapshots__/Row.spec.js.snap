// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 100%;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 3em;
}

.c0:not(:last-child) {
  padding-bottom: 1em;
}

.c0:not(:first-child) {
  padding-top: 1em;
}

@media (max-width:999px) {
  .c0 {
    -webkit-flex-wrap: wrap;
    -ms-flex-wrap: wrap;
    flex-wrap: wrap;
  }
}

<div
  className="c0"
/>
`;
